src/assets/styles/styles/custom/_dropdown.scss

```diff
@@ -0,0 +1,37 @@
+.dropdown-buttons-container {
+  background-color: white;
+  mix-blend-mode: multiply;
+  overflow: hidden;
+  border-bottom-left-radius: $btn-border-radius;
+  border-bottom-right-radius: $btn-border-radius;
+  border-top-left-radius: 0px;
+  border-top-right-radius: 0px;
+  box-shadow: 0 2.8px 2.2px $gray-400, 0 6.7px 5.3px $gray-400;
+  position: relative;
+  top: -63px;
+  right: -10px;
+  padding-bottom: 10px;
+  padding-top: 25px;
+  max-width: 119px;
+
+  button {
+    width: 119px;
```

Comment:
ustawiłam szerokość przycisków na maksymalną szerokość containera, więc zawsze będą zajmować cały dostępny obszar

Reply:
Myślę, że można by było zrobić to jako lokalna zmienną. 
Lub ustawić szerokość na 100%. 
Ale to na przyszłość, bo nie będę aż na tyle się czepiać :D